setup.py

```diff
@@ -86,6 +86,7 @@ def get_extensions():
     define_macros = []
 
     if int(os.environ.get('PYERFA_USE_SYSTEM_LIBERFA', 0)):
+        print('Using system liberfa')
```

Comment:
Was this just for debugging or deliberately here?

Reply:
Yes it is deliberate.
I prefer to have a track of what happens in the install log.